jekyll/_cci2/testing-ios.md

```diff
@@ -520,17 +501,12 @@ iPhoneSimulator: Timed out waiting 120 seconds for simulator to boot, current st
 cases CircleCI may not correctly detect the Xcode workspace, project, or
 scheme. Instead, you can specify these through environment variables.
 
-### Constraints on macOS-Based Builds
-There are a few features normally available on CircleCI's standard
-Linux containers that are not available for macOS builds at the moment:
+### Constraints on macOS-based Builds
+
+These are the features normally available on CircleCI 2.0
+Linux environment that are not available for macOS builds at the moment:
 
-* Parallelism is not supported
-* While the general `config.yml` file structure will be honored in macOS-based builds
-[configuration options]( {{ site.baseurl }}/2.0/configuration-reference/), the following sections of
-`circle.yml` will not work correctly:
-  * `machine: services`
-  * `machine: <language>`, where `<language>` is any language mentioned
-    in the [Configuration doc]( {{ site.baseurl }}/2.0/configuration-reference/)
+* Parallelism is not supported.
```

Comment:
Parallelism technically works, but there's no support for it inside Xcode... Might be better phrasing this as a section on "spitting up your builds" and including strategies for breaking up your tests into separate targets & jobs?